contracts/0.4.24/oracle/LidoOracle.sol

```diff
@@ -202,6 +250,7 @@ contract LidoOracle is ILidoOracle, AragonApp {
 
         uint256 i = 0;
         Report memory report = Report(_beaconBalance, _beaconValidators);
+        reportSanityChecks(report);  // rollback on boundaries violation
```

Comment:
Looks that we want to make this check 
- not to make it every `reportBeacon`, 
- but after the quorum is reached and we are ready to notify `Lido` contract.

Reply:
Hmmm, I thought the purpose of these boundaries is to deny "crazy" oracles to report and participate in quorum.
So they should be denied on the early stage. Am I right, @vshvsh?

Reply:
They are needed to protect lido from the wrong info, so only once on report beacon is good enough. Those checks are really expensive.